pkg/healthcheck/healthcheck.go

```diff
@@ -841,8 +991,11 @@ func expectedServiceAccountNames() []string {
 	}
 }
 
-func (hc *HealthChecker) checkClusterRoles() error {
-	crList, err := hc.kubeAPI.RbacV1().ClusterRoles().List(metav1.ListOptions{})
+func (hc *HealthChecker) checkClusterRoles(shouldExist bool) error {
+	options := metav1.ListOptions{
+		LabelSelector: k8s.ControllerNSLabel,
+	}
```

Comment:
It might be worth moving this into a global var given it's used multiple times.

Reply:
At this point, I prefer to keep this close to where it's used. As this is a big file, I'm reluctant to make this variable (which is used only by a handful of methods) a global, and am willing to live with a bit of redundancy.